src/controller/CHIPDeviceController.h

```diff
@@ -39,6 +39,7 @@
 #include <controller/CHIPDeviceControllerSystemState.h>
 #include <controller/CommissioneeDeviceProxy.h>
 #include <controller/CommissioningDelegate.h>
+#include <controller/CurrentFabricRemover.h>
```

Comment:
Why?  This should just be included in the .cpp

Reply:
I will change this include line to cpp file.